Mathlib/Order/OrderType.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) _
+-/
+import Mathlib.SetTheory.Cardinal.Basic
+import Mathlib.SetTheory.Ordinal.Basic
+import Mathlib.Order.Category.LinOrd
+import Mathlib.Algebra.Order.Ring.Unbundled.Rat
```

Comment:
In fact, maybe this hypothetical η-order file can have all of the material on ℚ as well?